spec/integration/query_result_builder_spec.rb

```diff
@@ -30,7 +30,7 @@ def self.id_namespace
       Object.send(:remove_const, :FooObject)
     end
     it "should return an array of objects that are of the class stored in active_fedora_model_s" do
-      query = "id\:#{RSolr.escape(@test_object.id)} OR id\:#{RSolr.escape(@foo_object.id)}"
+      query = "id\:#{ActiveFedora::SolrQueryBuilder.solr_escape(@test_object.id)} OR id\:#{ActiveFedora::SolrQueryBuilder.solr_escape(@foo_object.id)}"
```

Comment:
The query probably should be `SolrQueryBuilder.construct_query_for_ids([@test_object.id, @foo_object.id])`